src/uu/factor/src/cli.rs

```diff
@@ -59,9 +59,7 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
             }
         }
     } else {
-        let stdin = stdin();
-
-        for line in stdin.lock().lines() {
+        for line in stdin().lines() {
```

Comment:
Error here:
`use of unstable library feature 'stdin_forwarders'`

Reply:
I removed this fix - clippy is not 100% happy still, but at least it is much closer to passing now.

Reply:
Clippy should take the MSRV into account with the suggestions is makes, so I think we're good. We'll start seeing them once we bump the MSRV.